addons/mail_group/models/mail_group.py

```diff
@@ -251,6 +251,11 @@ def _alias_get_error_message(self, message, message_dict, alias):
         self.ensure_one()
 
         if alias.alias_contact == 'followers':
+            # Selected group of users
+            email = email_normalize(message_dict.get('email_from'))
```

Comment:
Probably would be good form to 
```suggestion
            email = email_normalize(message_dict.get('email_from', ''))
```